### PR TITLE
fix: Ensure that `getLastCrashReport()` is actually the last crash report

### PR DESCRIPTION
**Pull request by felixrieseberg**
This PR fixes https://github.com/electron/electron/issues/11749 by ensuring that `getLastCrashReport()` actually returns the last crash report (and not just a certain position in the array).

Also: A test that actually calls the method under test 😅

### CODE REVIEW

File: spec/api-crash-reporter-spec.js [hunk 1]
```diff
@@ -259,8 +259,11 @@ describe('crashReporter module', () => {
   describe('getLastCrashReport', () => {
     it('correctly returns the most recent report', () => {
       const reports = crashReporter.getUploadedReports()
-      const lastReport = reports[0]
+      const lastReport = crashReporter.getLastCrashReport()
+
+      // In our case, the first report is actually the newest
       assert(lastReport != null)
+      assert(lastReport === reports[0])
```
Comment (ckerr) [inline spec/api-crash-reporter-spec.js]: It's not clear from this block that this is testing the new code.

Would it be possible to add an `assert(reports.length > 1)` to ensure comparison happened?

**Comment by ckerr** [comment]
electron-linux-x64:
```text
AssertionError [ERR_ASSERTION]: false == true
    at Context.it (/home/builduser/project/spec/api-crash-reporter-spec.js:266:7)
```

electron-linux-ia32:
```text
not ok 987 crashReporter module getLastCrashReport correctly returns the most recent report
  AssertionError [ERR_ASSERTION]: false == true
      at Context.it (/var/lib/jenkins/workspace/ron_ARM64_electron_PR-12253-WP3JY47LEEJ2OR6WRBG4W43ZPAXD4YTS3ZGJCPY3GXIOPM3Q5EKQ@2/spec/api-crash-reporter-spec.js:266:7)
      at runCallback (timers.js:763:18)
      at tryOnImmediate (timers.js:734:5)
      at processImmediate (timers.js:716:5)
```

**Comment by MarshallOfSound** [comment]
@felixrieseberg Could you backport this to 1-8-x 😄 The bot did 2-0-x for you 👍

**Comment by ckerr** [comment]
@MarshallOfSound IMO `assert(reports.length > 1)` was still warranted and we're literally in the same TZ so :-1: on the dismissal.

Not a blocker though so ¯\_(ツ)_/¯ on this

**Pull request merged**